Fable.Remoting.MsgPack/Read.fs

```diff
@@ -335,24 +334,34 @@ type Reader (data: byte[]) =
 #else
             FSharpValue.MakeTuple (FSharpType.GetTupleElements t |> Array.map x.Read, t)
 #endif
+        elif t = typeof<DateTime> then
+            let dateTimeTicks = x.Read typeof<int64> :?> int64
+            let kindAsInt = x.Read typeof<int64> :?> int64
+            let kind =
+                match kindAsInt with
+                | 1L -> DateTimeKind.Utc
+                | 2L -> DateTimeKind.Local
+                | _ -> DateTimeKind.Unspecified
```

Comment:
Enums are supported so you can simply do
```fsharp
let kind = x.Read typeof<DateTimeKind> :?> DateTimeKind
```
and on the .NET write side (might have to move `writeEnum` higher up)
```fsharp
writeEnum dt.Kind out
```

It might be ever so slightly slower than what you did with explicit integer to enum mapping. Up to you ;)